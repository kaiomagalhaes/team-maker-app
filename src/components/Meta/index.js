import Head from 'next/head';
import { APP_NAME } from 'config';

const Meta = () => (
  <Head>
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta charSet="utf-8" />
    <link rel="shortcut icon" href="/static/favicon.png" />
    <link rel="stylesheet" type="text/css" href="/static/nprogress.css" />
    <title>{APP_NAME}</title>
  </Head>
);

export default Meta;
