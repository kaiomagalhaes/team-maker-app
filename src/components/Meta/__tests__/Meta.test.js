import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Meta from '../index';

describe('Meta', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Meta />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
