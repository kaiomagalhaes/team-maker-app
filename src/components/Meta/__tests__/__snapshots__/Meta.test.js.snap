// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meta renders and matches the snapshot 1`] = `
<SideEffect(Head)>
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <meta
    charSet="utf-8"
  />
  <link
    href="/static/favicon.png"
    rel="shortcut icon"
  />
  <link
    href="/static/nprogress.css"
    rel="stylesheet"
    type="text/css"
  />
  <title>
    APP NAME
  </title>
</SideEffect(Head)>
`;
